frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -149,49 +157,61 @@ const DatasetRow: FC<Props> = ({
     <StyledRow>
       <DetailsCell>
         <TitleContainer>
-          <StyledRadio
-            onChange={() => onSelect(dataset.id)}
-            checked={selected === dataset.id}
-          />
-          <div>{name}</div>
+          <span>{name}</span>
           {!isLoading && (
             <ErrorTooltip isFailed={isFailed} error={error} type={type} />
           )}
+          {isLoading && (
+            <AsyncUploadStatus
+              isWaiting={datasetStatus.upload_status === UPLOAD_STATUS.WAITING}
+              isConverting={
+                getConversionStatus(datasetStatus) ===
+                CONVERSION_STATUS.CONVERTING
+              }
+              isValidating={
+                datasetStatus.validation_status === VALIDATION_STATUS.VALIDATING
+              }
+              progress={datasetStatus.upload_progress}
+            />
+          )}
         </TitleContainer>
-        {isLoading && (
-          <AsyncUploadStatus
-            isWaiting={datasetStatus.upload_status === UPLOAD_STATUS.WAITING}
-            isConverting={
-              getConversionStatus(datasetStatus) ===
-              CONVERSION_STATUS.CONVERTING
-            }
-            isValidating={
-              datasetStatus.validation_status === VALIDATION_STATUS.VALIDATING
-            }
-            progress={datasetStatus.upload_progress}
-            datasetId={dataset.id}
-            collectionId={dataset.collection_id}
-          />
-        )}
       </DetailsCell>
       <Popover values={tissue} isLoading={isMetadataLoading} />
       <Popover values={assay} isLoading={isMetadataLoading} />
       <Popover values={disease} isLoading={isMetadataLoading} />
       <Popover values={organism} isLoading={isMetadataLoading} />
       <CellCount cellCount={cell_count} isLoading={isMetadataLoading} />
-      <ExploreButtonCell>
-        {hasCXGFile(dataset) && (
-          <AnchorButton
-            intent={Intent.PRIMARY}
-            outlined
-            text="Explore"
-            href={dataset?.dataset_deployments[0]?.url}
-            target="_blank"
-            rel="noopener"
-            data-test-id="view-dataset-link"
+      <ActionCell>
```

Comment:
This is the action cell, and we only show `MoreDropdown` (which has the delete action) given the right condition

<img width="1394" alt="Screen Shot 2021-03-31 at 12 20 53 PM" src="https://user-images.githubusercontent.com/6309723/113199084-8fd7f800-921b-11eb-9ca3-975187f86f05.png">